src/ape/cli/commands.py

```diff
@@ -23,6 +24,13 @@ class NetworkBoundCommand(click.Command):
     """
 
     def invoke(self, ctx: Context) -> Any:
+        # Check if the network_option is defined, if not, add it with default values
+        if not any(
+            isinstance(param, click.core.Option) and param.name == "network"
+            for param in self.params
+        ):
+            self.params.append(network_option())
```

Comment:
Usage should be updated. Every place we are using both `NetworkBoundCommand` and `network_option` in core should be able to be simplified now.

Examples:

1. `clid.md` guide
2. ape-cache CLI tools
3. ape-console CLI tools
4. ape-run CLI tools

and anywhere else you find it